Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -27,11 +27,7 @@ final class SitterProfileViewModel: ObservableObject {
     init() {
         currentSitter = loadSitterFromStorage()
 
-        name = currentSitter.name
-        surname = currentSitter.surname
-        phone = currentSitter.phone
-        bio = currentSitter.bio
-        pricePerHour = String(currentSitter.pricePerHour)
+        storeInitialValues(currentSitter)
```

Comment:
```suggestion
        setFieldsAccordingSitter(currentSitter)
```
What do you think about it?